# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/signaling/gtest/mediapipeline_unittest.cpp
# Commit: c7dbdd82e8e8
# Full Hash: c7dbdd82e8e8f9cf93fe92dca922755320f36b29
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 3
# Description:
#   Bug 1654112 - Set main thread as current TaskQueue when calling into libwebrtc so that audio calling works. r=bwc
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D102286
# ==============================================================================

diff -r 7e00d78d6389 -r c7dbdd82e8e8 media/webrtc/signaling/gtest/mediapipeline_unittest.cpp
--- a/media/webrtc/signaling/gtest/mediapipeline_unittest.cpp	Tue Feb 16 17:46:07 2021 +0100
+++ b/media/webrtc/signaling/gtest/mediapipeline_unittest.cpp	Mon Feb 01 09:28:23 2021 +0100
@@ -562,6 +562,7 @@
   }
 
  protected:
+  const TaskQueueWrapper::MainAsCurrent main_as_current_;
   const rtc::scoped_refptr<webrtc::SharedModuleThread> module_thread_;
   const webrtc::AudioState::Config audio_state_config_;
   const RefPtr<webrtc::AudioDecoderFactory> audio_decoder_factory_;
